examples/gno.land/p/demo/audio/bytebeat/bytebeat.gno

```diff
@@ -0,0 +1,70 @@
+package bytebeat
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"math"
+
+	"gno.land/p/demo/audio/biquad"
+	"gno.land/p/demo/audio/wav"
+	"gno.land/p/demo/ufmt"
+)
+
+func ByteBeat(seconds uint32, sampleRate uint32, bytebeat_func func(t int) int) (data string) {
```

Comment:
func doc? especially explaining how bytebeat_func should "transform" the input?

something more descriptive for `bytebeat_func`? what about `pitchFunc`?

Reply:
Much improved documentation here: e049239f